server/index.ts

```diff
@@ -29,11 +30,14 @@ headerSetup(app);
 // Define routes
 app.use("/signup", signupRouter);
 app.use("/login", loginRouter);
+app.use('/api', router);
```

Comment:
not sure this is needed as well

Reply:
actually I didnt add that row, I pulled it from main, so I don't know. I'll ask Eli